lib/auth/init.go

```diff
@@ -709,6 +710,21 @@ func generateAuthority(ctx context.Context, asrv *Server, caID types.CertAuthID)
 	return ca, nil
 }
 
+var secondFactorUpgradeInstructions = `
+Teleport v16 removed support for local authentication without a second
+factor enabled. The auth_service should be updated to enable it.
```

Comment:
```suggestion
Teleport requires second factor authentication for local users.
The auth_service configuration should be updated to enable it.
```

I prefer not to mention specific version numbers because they will go out of date.